### PR TITLE
Software development life cycle processes

### PR DESCRIPTION
**Pull request by TLeenders**
Hoi Bart, ik ben klaar met laatste versie van ons hoofdstuk. Wil jij er ook nog eens naar kijken?

### CODE REVIEW

**Review by rhpvorderman — DISMISSED** [review]
Very well-written and clear. I can work with this!

File: docs/fieldnorm/software_life_cycle_processes.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+and often find it difficult to translate these into well-defined technical requirements.
+
+Guidelines should support the following processes to gather, convert, track, and update the requirements during a project:
+1. Collaboration between end users and developers when defining software requirements. 
+2. Conversion of users' requirements into small, individual development tasks.
+3. A plan for keeping in touch with end users throughout the different stages of the development process 
+   to avoid unexpected adjustments to requirements later on.
+
+
+> 📝 _Note:
```
Comment (rhpvorderman) [inline docs/fieldnorm/software_life_cycle_processes.md]: The word Note is redundant here, as the text is already encased  in a box signifying a note. 

The same goes for the emoji. The text already stated that it is a note, and the box even moreso.

File: docs/fieldnorm/software_life_cycle_processes.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+The process follows the same procedures as the development, testing, and implementation phases. 
+Only certain maintenance processes that deviate from the normal procedures for development, testing, 
+and implementation should be documented (hotfixes, for example).
+
+## Hotfixes
+Hotfixes are updates which are directly uploaded in to the actual live software without going through the development, 
+testing and implementation phases of the SDLC. They are meant as a quick way to fix unforeseen but process breaking bugs. 
+In general, having well-defined development, testing and implementation processes would negate the need for hotfixes. However, there is always the possibility for bugs to occur. The laboratory should have clear guidelines on how and when hotfixes are performed. 
+
+> 📝 _Note: hotfixes could be classified as non-conforming work and might need to be followed up through the laboratories QMS, more on this in [user feedback](feedback.md)_
```
Comment (rhpvorderman) [inline docs/fieldnorm/software_life_cycle_processes.md]: See comment above

**Review by rhpvorderman — COMMENTED** [review]
Since I was reassigned, I did a re-review. It still looks good to me, although I did notice some nitpicks.

File: docs/fieldnorm/software_life_cycle_processes.md (hunk 2 above)
Comment (Bratdaking) [reply docs/fieldnorm/software_life_cycle_processes.md]: Yup we can remove both. However on the webpages the emoji looks nice, and in general looking around by similar pages emojis are added to a boxes to indicate whether it is additional information, a warning or for example an assignment. But if we do it here we should do this also on the other pages.

**Comment by tobias-beers** [comment]
Requirements has been added into this, Thomas, can you see if you agree with the end-result?

**Review by TLeenders — COMMENTED** [review]
looks good to me!

**Comment by TLeenders** [comment]
@rhpvorderman this request is read for the final review. @tobias-beers merged requirements into this part and I have removed the emoji's.

**Review by rhpvorderman — APPROVED** [review]
Looks good! I also like that the emojis are gone. There are much better alternatives:

dancing banana

**Pull request merged**